Clinical-Genomics/scout
# Images in variant page

This PR adds the ability to add custom images to STR variant page by associating an image with the HGNC id.

![image](https://user-images.githubusercontent.com/12696913/133590170-8cf9ed0b-c7b8-4f80-9cfb-aeeb2eaf129b.png)

**How to test**:
1. Load the demo case 643594 into the database.
2. Verify that STRs with repeat locus AFF2 has a new panel containing the custom images

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
